### ISSUE TITLE
Documentation for ktlint integration

### ISSUE THREAD

**Issue by schalkms** [arturbosch/detekt#925]
As discussed in #38

**Comment by jukkasi** [comment]
Tried to figure out how to get ktlint warnings by running detekt RC7-3 with no luck.

Running ktlint for the project gives me bunch of warnings, but running detekt for the same project does not give any formatter warnings.

**Comment by jukkasi** [comment]
So it works if I add in Gradle defaultProfile setting plugins = "lib/detekt/detekt-formatting.jar" (https://arturbosch.github.io/detekt/groovydsl.html#options-for-detekt-configuration-closure) but I'm not sure if this is the best way for this since I need to maintain detekt-formatting.jar manually

**Comment by arturbosch** [comment]
hey, thanks for testing this out.
The preferred way is just to add the dependency to the detekt configuration:
```
dependencies {
	detekt("io.gitlab.arturbosch.detekt:detekt-formatting:$usedDetektVersion")
}

```
I've just tested this and it works for detekt itself using the kotlin-dsl.
detekt-formatting docu and activating it for detekt's self analysis are my next two issues I will work one.
Stay tuned for the documentation or just ask here :)
